presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanVisitor.java

```diff
@@ -186,4 +188,9 @@ public R visitAssignUniqueId(AssignUniqueId node, C context)
     {
         return visitPlan(node, context);
     }
+
+    public R visitGroupReference(GroupReference node, C context)
```

Comment:
Why do we need this? My original intention was to not have to ever use visitors for GroupReference nodes and to eventually get rid of them altogether.